docker/go/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Dockerfile
+
+# Use the debian:testing-slim base image
+FROM debian:testing-slim
+
+RUN mkdir /src
+WORKDIR /src
+COPY src/ src
+
+# Install required packages
+RUN apt-get update && apt-get install -y \
```

Comment:
Can you try adding `apt-get upgrade` as well to update all the packages to the latest version. This should eliminate a lot of the debian "vulnerabilities"